source/geometry_model/two_merged_chunks.cc

```diff
@@ -0,0 +1,1090 @@
+              point2[1] = prm.get_double ("Chunk maximum longitude") * degtorad;
+              point3[1] = point1[1];
+              point4[1] = point2[1];
+              lower_repetitions[1] = prm.get_integer ("Longitude repetitions");
+              upper_repetitions[1] = lower_repetitions[1];
+
+              AssertThrow (point1[0] < point2[0],
+                           ExcMessage ("Inner radius must be less than outer radius."));
+              AssertThrow (point1[1] < point2[1],
+                           ExcMessage ("Minimum longitude must be less than maximum longitude."));
+              AssertThrow (point2[1] - point1[1] < 2.*numbers::PI,
+                           ExcMessage ("Maximum - minimum longitude should be less than 360 degrees."));
+              AssertThrow (point3[0] < point2[0],
+                           ExcMessage ("Grid merge radius must be less than outer radius."));
+            }
+
+          // Inform the manifold about the minimum longitude
+          manifold.set_min_longitude(point1[1]);
+          // Inform the manifold about the minimum radius
+          manifold.set_min_radius(point1[0]);
+          // Inform the manifold about the maximum depth (without topo)
+          manifold.set_max_depth(point2[0]-point1[0]);
+
+          if (dim == 3)
+            {
+              point1[2] = prm.get_double ("Chunk minimum latitude") * degtorad;
+              point2[2] = prm.get_double ("Chunk maximum latitude") * degtorad;
+              point3[2] = point1[2];
+              point4[2] = point2[2];
+              lower_repetitions[2] = prm.get_integer ("Latitude repetitions");
+              upper_repetitions[2] = lower_repetitions[2];
+
+              AssertThrow (point1[2] < point2[2],
+                           ExcMessage ("Minimum latitude must be less than maximum latitude."));
+              AssertThrow (point1[2] > -0.5*numbers::PI,
+                           ExcMessage ("Minimum latitude needs to be larger than -90 degrees."));
+              AssertThrow (point2[2] < 0.5*numbers::PI,
+                           ExcMessage ("Maximum latitude needs to be less than 90 degrees."));
+            }
+
+        }
+        prm.leave_subsection();
+      }
+      prm.leave_subsection();
+    }
+  }
+}
+
+// explicit instantiations
+namespace aspect
+{
+  namespace GeometryModel
+  {
+    namespace internal
+    {
+#define INSTANTIATE(dim) \
+  template class ChunkGeometry<dim>;
+      ASPECT_INSTANTIATE(INSTANTIATE)
+
+#undef INSTANTIATE
+    }
+
+    ASPECT_REGISTER_GEOMETRY_MODEL(TwoMergedChunks,
+                                   "chunk with lithosphere boundary indicators",
+                                   "A geometry which can be described as a chunk of a spherical shell, "
+                                   "bounded by lines of longitude, latitude and radius. "
+                                   "The radial boundaries have two boundary indicators, so the user "
```

Comment:
I find radial is not quite clear, it could mean boundaries of equal radius, or boundaries in radius direction.

```suggestion
                                   "The side boundaries have two boundary indicators, so the user "
```